src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java

```diff
@@ -96,16 +105,23 @@ public void onPlayerChat(PlayerChatEvent event) {
         if (currentServer.isEmpty()) return;
 
         String username = event.getPlayer().getUsername();
-        String server = currentServer.get().getServerInfo().getName();
         String content = event.getMessage();
 
-        String message = new StringTemplate(config.DISCORD_CHAT_MESSAGE)
-            .add("username", username)
-            .add("server", server)
-            .add("message", content)
-            .toString();
+        if (config.DISCORD_USE_WEBHOOKS) {
+            String avatar = String.format("https://crafatar.com/avatars/%s?overlay", event.getPlayer().getUniqueId());
```

Comment:
I was going to mention adding the `overlay` parameter, but you've already done that :)

Reply:
> Although, maybe the avatar URL should be configurable? In case the user has a preference of service.
> (with the player UUID being available as a variable in `config.toml`, a la `{username}` and `{message}` for `discord.chat.message`)

Should I add this? Would this belong in a kind of "expert" section of the config file?

-----

> On that note, I wonder if it's possible to get the avatar of players connected through [Geyser](https://github.com/GeyserMC/Geyser)?

This already works for me? Or is this just because of my configuration of Geyser (Bedrock players have to login with a Java account when joining my server).
![test](https://user-images.githubusercontent.com/45130096/198831056-b9a08abd-357f-4513-9379-c5e54e3b674b.jpg)

Reply:
since its relying on a third party service it should be configurable, as for being in a more advanced section of the config i dont mind either way

Reply:
> This already works for me? Or is this just because of my configuration of Geyser (Bedrock players have to login with a Java account when joining my server).

Likely because your Bedrock players are logging in with Java accounts, but I don't know the technical details.
Although I don't believe Bedrock edition skins show up on Java clients by default, so I'm not sure how that would work...

Reply:
I think resolving non java connected geyser players avatars should be a separate issue (if anyone actually needs it)